Use bundle version instead of latest as tag

vkmc opened the pull request:
And use external registry instead of internal when creating the index image with bundles (no need to use the internal one)

csibbitt commented:
"And use external registry instead of internal " - I'm not seeing this in the changes, what am I missing?

Change otherwise makes sense to me.

vkmc commented:
Sorry about that, I reverted the change and fixed the tag there as well. Now we point to the internal registry with the correct tag. So this works with either deploy from bundles + index and local build + index.
